tests/device.test.ts

```
@@ -69,18 +69,30 @@ describe.concurrent('[GET] /device operations', () => {
+    for (let attempt = 0; attempt < 5; attempt++) {
+      response = await fetch(`${BASE_URL}/device`, {
+        method: 'POST',
+        headers,
+        body: JSON.stringify({
+          app_id: APPNAME_DEVICE,
+          device_id: deviceId,
+          channel: 'no_access',
+        }),
+      })
+
+      if (response.status === 200) break
```

ESLint

[error] Expect newline after if

(antfu/if-newline)